include/ginkgo/core/log/stream.hpp

```diff
@@ -178,6 +178,7 @@ class Stream : public Logger {
      * @param verbose  whether we want detailed information or not. This
      *                 includes always printing residuals and other information
      *                 which can give a large output.
+     * @param prefix  string to prefix the log message with.
```

Comment:
I think all the changes in this file should be part of the distributed stream logger PR.